.bumpversion.cfg

```diff
@@ -32,3 +32,8 @@ search = version = \"{current_version}\"
 [bumpversion:file:bindings/rust/evmc-vm/Cargo.toml]
 search = version = \"{current_version}\"
 
+[bumpversion:file:docs/EVMC.md]
+parse = **ABI Version (?P<major>\d+)**
+search = **ABI Version {major}**
```

Comment:
I think you need to do it exactly like in `[bumpversion:file:include/evmc/evmc.h]`
```
serialize = {major}
search = **ABI Version {current_version}**
replace = **ABI Version {new_version}**
```

Reply:
Skip `**` in search?